src/tests/ftest/container/attribute.py

```diff
@@ -116,7 +116,7 @@ def create_data_set(self):
     def test_container_attribute(self):
         """
         Test basic container attribute tests.
-        :avocado: tags=container,container_attr,attribute,sync_conattribute
+        :avocado: tags=all,tiny,full_regression,container,container_attr,attribute,sync_conattribute
```

Comment:
Did we decide on actually populating an _all_ tag (into every test?) rather than letting launch.py just include every test when given an *all* tag to filter on?

Reply:
So its either pr or full_regression, with the idea that if its a really great test that is going to catch any backsliding it goes in pr and everything else goes in full_regression but they go into one or the other.